### ISSUE TITLE
Re-enable Travis cache after 1037 is merged

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#1038]
### Problem

I upgraded our `package-lock.json` to a new format (via node 15) in #1037, but because of this had to modify `.travis.yml` to invalidate the cache.

### Solution

After #1037 is merged, send in another PR to re-enable Travis caching by modifying `.travis.yml` to:

```yml
language: node_js
node_js:
  - "15"
dist: trusty
compiler: clang # per: https://docs.travis-ci.com/user/languages/c
env:
  - NO_PERSIST
  - GI_PERSIST
branches:
  only:
    - master # https://github.com/okTurtles/group-income-simple/issues/58
cache:
  # Caches $HOME/.npm when npm ci is default script command
  # Caches node_modules in all other cases
  npm: true
  directories:
    - ~/.npm # NPM packages 
    - ~/.cache # Cypress binary
  # https://cypress.slides.com/cypress-io/cypress-on-ci#/9/7/0
  override:
    - npm ci
    - npm run cy:verify
install:
  - npm ci
script:
  - npm test
```
